layouts/almalinux-day-germany-2024/single.html

```diff
@@ -393,7 +393,7 @@ <h4 style="font-size: 23px; letter-spacing: 0.80px; font-weight: 500; margin-bot
             </div>
             <hr size=3px width=75% align="left" color="white" style="margin-top: 5px; margin-bottom: 5px;">
             <span style="letter-spacing: 0.80px;">
-              {{ i18n "11:15 am" }}<br>
+              {{ i18n "11:00 am" }}<br>
```

Comment:
Is this something that needs to be localized?

Reply:
I think yes, since there's always the possibility that other languages represent numbers differently than here. ([this is the answer](https://www.quora.com/Do-numbers-look-the-same-in-every-language) that made me decide to localize them)

Reply:
Nice read! I hadn't thought about that. Good call :).